docs/catalyst_101.md

```diff
@@ -73,14 +73,14 @@ After taxes, contractors and members receive the same hourly rate. Employee-memb
-| **Benefits**         | - [$150 healthcare stipend](policies.md#health-care)<br>                                                                                                                                                                                                                                                                                                                                                                          | All contractor benefits, plus:<br>- **[IRA account and retirement contributions](policies.md#retirement-accounts)**<br>- **[$3000 / 3 year depreciating tech stipend](policies.md#home-office-reimbursements)**<br>- **[$200 / year last mile reimbursements](policies.md#catalyst-irl-reimbursement) for visiting other members**<br>- **[1 hr paid time off / 52 work hours](policies.md#paid-time-off)**<br>- **Member capital account based on co-op patronage**<br>- **[Up to six months unpaid time off with adequate notice](policies.md#time-off--reduced-hours)**                                                                                                                       |
-| **Responsibilities** | Contractors are involved in everything except co-op governance decisions. They participate in all regular co-op meetings, including the financial check-ins and business teams after 3 months. Contractors can take as much time off as they want so long as they are considering the technical membership eligibility requirement of 500 hours over the course of six months and the social, unwritten requirement of not randomly disappearing. | Member-employees wear many hats. Until we have enough members to determine a cap, all members become part of the Catalyst board. Both members and board members have a say in co-op governance issues and are expected to act in the interest of the co-op. Members are expected to participate in meetings and engage in the technical and business aspects of the co-op. |
+|                      | **Contractors**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | **Employee-Members**                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
```

Comment:
@bendnorman This is the whitespace I was talking about.

Reply:
Or at least it showed up as whitespace in the split diff view 🤷

Reply:
This is the format the markdown pre-commit hook came up with. Markdown tables are pretty messy.